lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -211,12 +211,16 @@ def self.host
         end
 
         def self.api_host
+          # only used for deployment task
           Proc.new do
```

Comment:
jsyk, we're about to enable a Rubocop linter that prefers `proc` over `Proc.new`, but this totally doesn't have to be changed now.

Reply:
This line isn't new, so it's actually already been changed in the proc pr! I wont change it here so we can avoid merge conflicts.